spec/abstractops.html

```diff
@@ -1461,21 +1461,19 @@ <h1>
     <emu-alg>
       1. If ParseText(StringToCodePoints(_isoString_), |TemporalInstantString|) is a List of errors, throw a *RangeError* exception.
       1. Let _result_ be ? ParseISODateTime(_isoString_).
-      1. Let _offsetString_ be _result_.[[TimeZone]].[[OffsetString]].
-      1. If _result_.[[TimeZone]].[[Z]] is *true*, then
-        1. Set _offsetString_ to *"+00:00"*.
-      1. Assert: _offsetString_ is not *undefined*.
+      1. Assert: Either _result_.[[TimeZone]].[[OffsetString]] is not *undefined* or _result_.[[TimeZone]].[[Z]] is *true*, but not both.
```

Comment:
The fewer levels of negation, the easier to understand? ("is a String" avoids the awkward double negation of "X is not undefined"; I thought about reversing the XOR condition to "X is undefined or Y is false" but that's still kind of a double negation)
```suggestion
      1. Assert: Either _result_.[[TimeZone]].[[OffsetString]] is a String or _result_.[[TimeZone]].[[Z]] is *true*, but not both.
```